internal/pkg/service/buffer/store/export.go

```diff
@@ -32,6 +32,8 @@ func (s *Store) CreateExport(ctx context.Context, export model.Export) (err erro
 		s.createExportBaseOp(ctx, export.ExportBase),
 		s.createMappingOp(ctx, export.Mapping),
 		s.createTokenOp(ctx, model.TokenForExport{ExportKey: export.ExportKey, Token: export.Token}),
+		s.createFileBaseOp(ctx, export.File.FileBase),
+		s.createSliceBaseOp(ctx, export.File.CurrentSlice.SliceBase),
```

Comment:
@michaljurecko Když nebude `Slice` ve `File`, tak jak ho mám předat tady to tý transakce k vytvoření? Nevím jestli ho můžu dát přímo do `Export`, když je navázaný na `fileID`.

Reply:
:eyes:

Reply:
@JakubMatejka pri vytvoreni exportu sa ma v transakcii vytvorit prvy file a prva slice.
Je tam nejaky problem?

Dalsie files/slices bude riesit uz worker, to ta tu netrapi.

Reply:
Za mna je to v pohode.